### DISCUSSION TITLE
Made small fixes to the code

### DISCUSSION THREAD

**Discussion by tanmaypardeshi** [tanmaypardeshi/CSC-510-Project3-PopcornPicks#14] [General]
## Made the following changes in this PR:

1. Added the flask_login and flask_db to improve the overall experience with db.
2. Slightly changed the login and signup flow to avoid the use of ajax unecessarily.
3. Refactored the code to make it more structured.
4. Made changes to the README for steps  to set the database file up.

## Things left to do:

1. Improve the unit tests

Please review my code and then leave your comments here. If there is nothing to change, please merge my code. To the main branch.

**Comment by ExtremeMachine12** [comment]
Mostly good, but I recommended a few changes so we pass our workflows before merging.

**Comment by tanmaypardeshi** [comment]
I have updated the workflows and fixed them. Closing this discussion as it is fixed.
